### ISSUE TITLE
Comma

### ISSUE THREAD

**Issue by ltaback20** [google/codeworld#925]
Code World believes that I have a comma in one of the spots but I do not. I even tried deleting it then re-writing it but that still didn't work. My friend is experiencing the same problem please help!

**Shared Program**: https://code.world/#Pxxxxxxxxxxx (if applicable)

**Description**

(If you don't know what's causing the problem, please include the browser and operating system you're using in the description.)

**Comment by cdsmith** [comment]
I don't understand what you mean.  It would help a lot if you post a link to the code that's causing the problem., or give steps to try this out and see the bug.

**Comment by cdsmith** [comment]
It looks like this might be related to the unhelpful-error report I also got for https://code.world/#PBY63KHtKyldu88U1KwvCxg.  The problem here is that there's an extra comma at the end of the `scaled` function.

The error is pointing at almost the right place, but it targets the beginning of the tuple, rather than the location of the comma itself.  I filed bug #926 earlier with an idea for how to give a more accurate location in the error message.
